PEAR.php

```diff
@@ -508,7 +508,7 @@ function delExpect($error_code)
      * @see PEAR::setErrorHandling
      * @since PHP 4.0.5
      */
-    function &raiseError($message = null,
+    static function raiseError($message = null,
```

Comment:
It looks like upstream PEAR 1.10.0 has been patched with a clever work-around -- for any functions which were (by design) allowed to be static or non-static, these have been converted to magic-methods. I'll add the upgrade to this PR.